Panics when starting a vm

## The problem

Failed to start a vm by using the latest sdk.

## Environment

* Desktop OS/version: macOS High Sierra
* Java version to run maven: oracle-jdk-1.8.0_92
* Go version under test: 1.10.1
* The metamodel version speicifed in the root pom.xml: 1.3.0
* The model version sepecified in the root pom.xml: 4.3.7

## Description

When I starting a vm with codes: 

```go
vmsService.VmService(vmID).Start().Send()
```

it panics with outputs as blow

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x133dfce]

goroutine 1 [running]:
gopkg.in/imjoey/go-ovirt%2ev4.(*vmServiceStartRequest).Send(0xc42012df10, 0x0, 0x0, 0x0)
	/Users/joey/Work/golangspace/src/gopkg.in/imjoey/go-ovirt.v4/services.go:62078 +0x19e
```
**Reproduces how often:** 100%.

--- a/generator/src/main/java/org/ovirt/sdk/go/ServicesGenerator.java
+++ b/generator/src/main/java/org/ovirt/sdk/go/ServicesGenerator.java
@@ -497,7 +497,9 @@ public class ServicesGenerator implements GoGenerator {
                 String paraArgName = goNames.getParameterStyleName(parameter.getName());
                 String paraMethodName = goNames.getExportableMemberStyleName(parameter.getName());
                 if (goTypes.isGoPrimitiveType(parameter.getType())) {
-                    buffer.addLine("actionBuilder.%1$s(*p.%2$s);", paraMethodName, paraArgName);
+                    buffer.addLine("if p.%1$s != nil {", paraArgName);
+                    buffer.addLine("  actionBuilder.%1$s(*p.%2$s);", paraMethodName, paraArgName);
+                    buffer.addLine("}");
                 } else {
                     buffer.addLine("actionBuilder.%1$s(p.%2$s);", paraMethodName, paraArgName);
                 }

